From b0d31cc1e9a43b45d216ee7f804e901a5c0f8936 Mon Sep 17 00:00:00 2001
From: Hodorgasm <nsane457@gmail.com>
Date: Mon, 19 Sep 2016 17:58:22 -0400
Subject: [PATCH] Fix "stdlib.h: No such file or directory" errors in GCC-6

Including '-isystem /usr/include' breaks building with GCC-6.
See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=70129

Upstream-Status: Accepted [https://github.com/ros/geometry/commit/b0d31cc1e9a43b45d216ee7f804e901a5c0f8936]

This patch has been generated with:
  `git format-patch -1 b0d31cc1e9a43b45d216ee7f804e901a5c0f8936`
in the geometry repository.

Signed-off-by: Lukas Bulwahn <lukas.bulwahn@oss.bmw-carit.de>
---
 tf/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tf/CMakeLists.txt b/tf/CMakeLists.txt
index 8077561..cf72127 100644
--- a/tf/CMakeLists.txt
+++ b/tf/CMakeLists.txt
@@ -8,7 +8,7 @@ find_package(catkin COMPONENTS angles geometry_msgs message_filters message_gene
 
 catkin_python_setup()
 
-include_directories(SYSTEM ${Boost_INCLUDE_DIR}
+include_directories(${Boost_INCLUDE_DIR}
                            ${catkin_INCLUDE_DIRS}
 )
 include_directories(include)
-- 
2.5.5

